### ISSUE TITLE
authenticate requests to reports#index

### ISSUE THREAD

**Issue by rurabe** [rurabe/ihs-server#1]
do they need user management? or is basic auth sufficient?

**Comment by randalt** [comment]
Talked to Kimo and he said one general account for IHS and another for the State will be fine for now. It's possible that further down the line that this can be deployed to other organizations.

**Comment by rurabe** [comment]
yeah, was thinking something pretty basic for the mvp and demo, but eventually they will need user accounts
